packages/components/src/internal/components/search/utils.ts

```diff
@@ -375,11 +423,12 @@ export function getUpdateFilterExpressionFilter(
             if (clearBothValues) {
                 value = null;
             } else if (isSecondValue) {
-                if (!newFilterValue) value = previousFirstFilterValue ? previousFirstFilterValue : '';
-                else value = (previousFirstFilterValue ? previousFirstFilterValue + ',' : '') + newFilterValue;
+                if (newFilterValue == null) value = previousFirstFilterValue != null ? previousFirstFilterValue : '';
+                else value = (previousFirstFilterValue != null ? previousFirstFilterValue + ',' : '') + newFilterValue;
```

Comment:
I'm not sure, but I wonder if this causes problem for the "[blank]" or "[All]" values.